newsletter-november-2023.rst

```diff
@@ -0,0 +1,44 @@
+.. post:: Nov 3, 2023
+   :tags: newsletter, python
+   :author: Eric
+   :category: Newsletter
+
+Read the Docs newsletter - November 2023
+========================================
+
+
+News and updates
+----------------
+
+* Work continues on hardening `Addons <https://github.com/readthedocs/addons/>`_, our new in-documentation JavaScript client that supports all documentation tools. We're looking for people in the community to test out this new functionality, and will be expanding access in the near future.
+* **Python 3.12** is now supported on builds, and is the default version used when you specify ``build.tools.python: 3`` in your configuration file.
+* Language codes are now normalized in URLs: https://blog.readthedocs.com/language-codes-are-now-normalized/.
+* Our search infrastructure was upgraded to Elastic Search 8, which should provide faster and better search results.
+* We `submitted a fix <https://github.com/florimondmanca/djangorestframework-api-key/pull/244>`_ to the API Key library we use to make API calls to our servers much faster, which should result in faster build times for all users.
+
+
+Upcoming changes
+----------------
+
+* We are working to expand the functionality of our `redirects feature <https://github.com/readthedocs/readthedocs.org/pull/10825>`_ to support more use cases. More will be announced here in the coming month.
+* We are planning an upgrade to our dashboard notification system, so that users have more control and better context for on-site notifications.
```

Comment:
```suggestion
* We are planning an `upgrade to our dashboard notification system <https://github.com/readthedocs/readthedocs.org/pull/10890>`_, so that users have more control and better context for on-site notifications.
```